test/modules/Modules.Test/TestResultCoordinator/Reports/CountingReportGeneratorTest.cs

```diff
@@ -60,6 +61,7 @@ public void TestConstructorSuccess()
 
             var reportGenerator = new CountingReportGenerator(
                 TestDescription,
+                TestMode.Connectivity,
```

Comment:
Please consider doing `TestMode.E2E` instead of `TestMode.Connectivity`. 

```
namespace TestResultCoordinator
{
    enum TestMode
    {
        Connectivity,
        E2E = Connectivity,
        LongHaul
    }
}
```
PS. please check if that's legal in Dotnet3.1 syntax

Reply:
What is the motivation for this change?

Reply:
It sounds a little odd that `Connectivity` mode is used in `E2E` test.

Reply:
This is a unit test for the TRC, not a part of the E2E test suite. Unless I am misunderstanding what you thought. The paths look similar.